src/User.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+const propTypes = {
+  children: PropTypes.element,
+};
+
+function User({ children }) {
```

Comment:
The form should be UserForm to be more explicitly separate from the User profile card